project/plugins.sbt

```diff
@@ -7,7 +7,7 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
 
```

Comment:
auto-build should be set to 3.0.0 and then build.sbt cleaned as per the notes in the epic ticket